modules/abstract-cosmos/src/lib/transaction.ts

```diff
@@ -22,15 +22,18 @@ import {
   TxData,
   WithdrawDelegatorRewardsMessage,
 } from './iface';
-import utils from './utils';
+import { CosmosUtils } from './utils';
 
 export class CosmosTransaction extends BaseTransaction {
   protected _cosmosLikeTransaction: CosmosLikeTransaction;
   protected _accountNumber: number;
   protected _chainId: string;
 
-  constructor(_coinConfig: Readonly<CoinConfig>) {
+  protected _utils: CosmosUtils;
```

Comment:
Can we use the utils file without having to pass it to the constructor ? Also if we are using CosmosUtils for all of the chains, then do we really need to pass it ?

Reply:
Just looked, seems like we have side chain specific utility methods which extend from cosmos utils, so I'm assuming we're passing coin specific utility methods. Ignore the above comment in that case